# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/WorkerScope.h
# Commit: 5278e40b80c3
# Full Hash: 5278e40b80c3c4aa642a24a1ff5a50247b9b3126
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2022-11-28 15:41:50
# Regressor Bug: 1778510
# File Overlap Count: 1
# Description:
#   Bug 1778510: Add RTPCallerType to GlobalObject r=asuth
#   
#   This centralizes the logic in one place.
#   
#   In order to do this, we will need to check the principal
# ==============================================================================

diff -r dba220581d93 -r 5278e40b80c3 dom/workers/WorkerScope.h
--- a/dom/workers/WorkerScope.h	Mon Nov 28 04:21:26 2022 +0000
+++ b/dom/workers/WorkerScope.h	Mon Nov 28 04:21:26 2022 +0000
@@ -173,6 +173,8 @@
  protected:
   ~WorkerGlobalScopeBase();
 
+  bool IsSystemPrincipal() const final;
+
   CheckedUnsafePtr<WorkerPrivate> mWorkerPrivate;
 
   void AssertIsOnWorkerThread() const {